.github/workflows/weekly-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Weekly
+
+on:
+  schedule:
+    - cron: "0 0 * * 6" # Runs every saturday at midnight
+  pull_request:
+    paths:
+      - .github/workflows/weekly-test.yaml
+permissions:
+  contents: read
+  actions: read
+
+jobs:
+  cncf:
+    name: CNCF conformance test
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04"]
+        arch: ["amd64"]
+        channel: ["latest/edge"]
```

Comment:
Is this correct? Where is the channel used? I would expect this to be 1.32-classic/stable